.github/workflows/docker.yml

```diff
@@ -30,8 +42,9 @@ jobs:
     - name: Build and push
       uses: docker/build-push-action@v2
       with:
+        platforms: ${{ matrix.platform }}
```

Comment:
Is this right to be doing a matrix build/push for each platform? Do new platforms push into existing images? 

If not, remove the matrix from this job and add a `platforms` output. Then, split `docker run` out into a `test` job. That job will be a matrix job, using the `platforms` output from the build job to set the `platform` list for the matrix.

Reply:
Good question. After looking more closely with `docker image inspect`, this might not be working the way I'd assumed. 😅